### DISCUSSION TITLE
how do I send the rich text to form post?

### DISCUSSION THREAD

**Discussion by matiazar** [slab/quill#4336] [Q&A]
how do I send the rich text to form post?
this rich text is inside a div with an id
how do I send as an input field?

thanks in advance!

**Comment by xsalagarcia** [comment]
Hi,

you can select your form and set an event [formdata](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/formdata_event). This event is triggered after the entry list representing the form's data is constructed (when form is submitted (or used in the FormData object creation), but before is sent. 

You can see the example here [https://quilljs.com/playground/form](https://quilljs.com/playground/form)
Select the example "Using quill inside a form" and look at the code.

**Comment by matiazar** [comment]
thanks!
